### PR TITLE
Edge13 ES6 + ES7 tables updated (build 10576)

### PR DESCRIPTION
**Pull request by sarahkadar**

### CODE REVIEW

File: data-es6.js [hunk 1]
```diff
@@ -11986,6 +12031,7 @@ exports.tests = [
     rhino17:     true,
     node012:     true,
     android40:   true,
+    edge13:      false,
```
Comment (zloirock) [inline data-es6.js:12034]: Emm... Seriously Edge 13 does not support HTML-style comments?

Comment (sarahkadar) [inline data-es6.js:12034]: well, that's what Edge 13 latest tells me. Either that, or the test is faulty somehow.

Comment (zloirock) [inline data-es6.js:12034]: Yep. Edge 12 and IE also throws on the first line of test.

File: data-es7.js [hunk 2]
```diff
@@ -172,6 +161,23 @@ exports.browsers = {
...
+    short: 'Edge 12',
+    note_id: 'edge-experimental-flag',
+    note_html: 'Flagged features have to be enabled via "Enable experimental Javascript features" setting under about:flags'
+  },
+  edge13: {
+    full: 'Internet Explorer, Microsoft Edge for preview Builds',
+    short: 'Edge 13',
+    note_id: 'edge-experimental-flag',
+    note_html: 'Flagged features have to be enabled via "Enable experimental Javascript features" setting under about:flags'
+  },
```
Comment (zloirock) [inline data-es7.js]: Please, restore columns order for consistency ES6 table. Realy strange see IE and Edge between Node 0.12 and Node 4 :)

File: data-es7.js [hunk 3]
```diff
@@ -1233,7 +1275,7 @@ exports.tests = [
       */},
       res: {
         babel:      true,
-        edge:       true,
+        edge12:       true,
         firefox31:  true,
```
Comment (zloirock) [inline data-es7.js:1279]: Please, fix alignment everywhere.

File: data-es7.js (hunk 2 above)
Comment (sarahkadar) [inline data-es7.js]: done and sorry

File: data-es7.js (hunk 3 above)
Comment (sarahkadar) [inline data-es7.js:1279]: done and sorry, hope its okay now. also fixed some that were unaligned the first place.

**Comment by zloirock** [comment]
Thanks.

**Pull request merged**